tools/src/main/java/org/apache/kafka/trogdor/coordinator/TaskManager.java

```diff
@@ -127,7 +127,12 @@
      */
     private long nextWorkerId;
 
-    TaskManager(Platform platform, Scheduler scheduler, long firstWorkerId) {
+    /**
+     * The TrogdorMetrics for this coordinator.
+     */
+    private TrogdorMetrics trogdorMetrics;
```

Comment:
This should be final and probably just be called "metrics" since there are no other metrics that we need to distinguish it from